### ISSUE TITLE
blinks in Pupil Player

### ISSUE THREAD

**Issue by mkassner** [pupil-labs/pupil#968]
we need ab able to detect and csv export them in Player.

**Comment by RSKothari** [comment]
Intermediate solution until fix comes out:

```
import msgpack
import scipy.io as io
fid = open('pupil_data', 'rb')
pupil_data = msgpack.unpack('pupil_data', encoding='utf-8')
io.savemat('pupil_data', pupil_data) # Warning: takes a lot of time

list_blinks = pupil_data['blinks']
first_blink = list_blinks[0]
first_blink_duration = first_blink['base_data'][-1]['timestamp'] - first_blink['base_data'][0]['timestamp']
```

**Comment by papr** [comment]
**This code example will override your pupil_data file and make it impossible to open the recording as it was in Pupil Player again!** Use with care. Change line `io.savemat(...)` appropriately to save the Matlab matrices in a different file.

Also, I am pretty sure that that one has to pass `fid` to `msgpack.unpack` instead of `'pupil_data' ` to unpack the original file.

My suggestion would be to remove the Matlab data conversion part, process the data in Python as the last  5 lines do, and save the extracted data with the Python `csv` module.

Be aware that this only works if you had activated the Online Blink Detection plugin during the recording in Capture.

**Comment by mkassner** [comment]
@papr  adding blink detection in Pupil Player should not be hard. I think we should add this instead of showing recorded blinks. @papr  what do you think?

**Comment by papr** [comment]
Closed with #1040
